lib/node/runner/checker_message.go

```diff
@@ -158,7 +158,7 @@ func PushIntoTransactionPoolFromNode(c common.Checker, args ...interface{}) erro
 		return err
 	}
 
-	checker.Log.Debug("push transaction into TransactionPool from node", "transaction", tx)
+	checker.Log.Debug("push transaction into TransactionPool from node")
```

Comment:
How about `from node","tx",tx.Hash)`?

Reply:
@anarcher This ` checker.Log` already have, https://github.com/bosnet/sebak/blob/c3199102056466cbad966bcfbc6de3f08f6b2428/lib/node/runner/checker_message.go#L58 .